app/views/support/LandingPageOps.scala

```diff
@@ -0,0 +1,35 @@
+package views.support
+
+import com.gu.memsub.promo.Promotion.{AnyPromotion, AnyPromotionWithLandingPage}
+import com.gu.memsub.promo._
+
+object LandingPageOps {
+
+  private def getSectionColour(promotion: AnyPromotionWithLandingPage) = promotion.landingPage.sectionColour.getOrElse(Blue)
+
+  implicit class ToLandingPageClassName(promotion: AnyPromotionWithLandingPage) {
+    def landingPageSectionColour: String = {
```

Comment:
this is a tiny tiny thing but I think you can have one implicit class with multiple methods rather than an implicit class per method